functional_widget/lib/src/parameters.dart

```diff
@@ -37,6 +37,9 @@ Parameter _parseParameter(ParameterElement parameter) {
   return Parameter(
     (b) => b
       ..name = parameter.name
+      ..defaultTo = parameter.defaultValueCode != null
```

Comment:
These changes shouldn't be in that PR.
You may want to revert d1d05130e72cfba928fabe9d772f1b1d68c08c9d

Reply:
Should be gone after rebasing.